.TestDockerfile

```diff
@@ -14,4 +13,4 @@ ADD dev-requirements.txt /app/
 RUN pip install -r /app/dev-requirements.txt
 ADD carto_renderer /app/carto_renderer
 
-RUN PYTHONPATH=/app py.test -vv /app/carto_renderer
+RUN PYTHONPATH=/app py.test -vv -W ignore::DeprecationWarning /app/carto_renderer
```

Comment:
could remove this if it's a really bad idea. An test dependency was using a deprecated feature. The dep was updated a few days ago but not released yet: https://github.com/pytest-dev/pytest-asyncio/issues/141

Reply:
I'd recommend waiting for the dep to update.

Reply:
KIADA!!!

Reply:
hi Kaida!

Reply:
Hi!!!